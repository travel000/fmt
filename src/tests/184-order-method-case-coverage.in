<?php
//passes:TwoCommandsInSameLine,OrderAndRemoveUseClauses,AddMissingCurlyBraces,SmartLnAfterCurlyOpen,NormalizeLnAndLtrimLines,MergeParenCloseWithCurlyOpen,MergeCurlyCloseAndDoWhile,MergeDoubleArrowAndArray,ExtraCommaInArray,ResizeSpaces,Reindent,ReindentColonBlocks,ReindentObjOps,EliminateDuplicatedEmptyLines,AlignEquals,AlignDoubleArrow,RTrim,OrderMethod,NormalizeLnAndLtrimLines,Reindent


abstract class A {
	public function setA($a){
		$this->a = $a;
	}
	abstract public function b();
}
