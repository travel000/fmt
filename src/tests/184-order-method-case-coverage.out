<?php
//passes:TwoCommandsInSameLine,OrderAndRemoveUseClauses,AddMissingCurlyBraces,SmartLnAfterCurlyOpen,NormalizeLnAndLtrimLines,MergeParenCloseWithCurlyOpen,MergeCurlyCloseAndDoWhile,MergeDoubleArrowAndArray,ExtraCommaInArray,ResizeSpaces,Reindent,ReindentColonBlocks,ReindentObjOps,EliminateDuplicatedEmptyLines,AlignEquals,AlignDoubleArrow,RTrim,NormalizeLnAndLtrimLines,Reindent,OrganizeClass

abstract class A {
	abstract public function b();

	public function setA($a) {
		$this->a = $a;
	}
}
